Makefile

```diff
@@ -9,15 +9,18 @@ endif
 
 ifeq ($(ROCKSDB_CHECK), 0)
 	TAGS += rocksdb
-endif 
+    CGO_CXXFLAGS := "${CGO_CXXFLAGS} -std=c++11"
+    CGO_LDFLAGS := "${CGO_LDFLAGS} -g -O2 -lrocksdb -lstdc++ -lm -lz -lbz2 -lsnappy -llz4 -lzstd"
```

Comment:
can we remove `-lrocksdb -lm -lz -lbz2 -lsnappy -llz4 -lzstd` here? 

above check has already checked `-lrocksdb` and for the following links, I prefer passing outside the Makefile manually.

Reply:
Yes, It seems that it's OK to remove it.